scripts/update_review_app.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
`#!/usr/bin/env bash` is usually more portable than expecting a specific system path.

Reply:
Going to pass on this for now. Did some quick reading and realized it seems like there are pros and cons on both sides here. At this point, the goal of the script isn't to be portable across systems, but support the specific Docker container running in Circle